Workbooks/Communication Services/Email_Insights/Email_Insights.workbook

```diff
@@ -0,0 +1,983 @@
+{
+    "version": "Notebook/1.0",
+    "items": [
+      {
+        "type": 11,
+        "content": {
+          "version": "LinkItem/1.0",
+          "style": "tabs",
+          "links": [
+            {
+              "id": "8e2d5862-e7d0-4238-9b61-b1e2b7a0b541",
+              "cellValue": "tab",
+              "linkTarget": "parameter",
+              "linkLabel": "Email Overview",
+              "subTarget": "stats",
+              "style": "link",
+              "linkIsContextBlade": true
+            },
+            {
+              "id": "7667d622-e3a3-46f9-98b4-ecd7f4deee3d",
+              "cellValue": "tab",
+              "linkTarget": "parameter",
+              "linkLabel": "Email Performance",
+              "subTarget": "perf",
+              "style": "link"
+            }
+          ]
+        },
+        "name": "links - 4"
+      },
+      {
+        "type": 9,
+        "content": {
+          "version": "KqlParameterItem/1.0",
+          "parameters": [
+            {
+              "id": "1a744cde-5e97-45e8-9966-ccef8b6a698d",
+              "version": "KqlParameterItem/1.0",
+              "name": "Time",
```

Comment:
You may also want to study having like in Overview notebook a second time dimention for aggregation/granularity 

As currently you picked 4h as the default bin size, but if in the future a customer wants to pick a time in workbook spanning a long period, some customer may want to be able to configure "daily" numbers vs 4h bins.

Reply:
I added a time bucket dropdown and updated the charts
![image](https://github.com/microsoft/Application-Insights-Workbooks/assets/89162957/c5d5359f-a502-432f-8d43-caddf98e575d)